src/routes/courses/[slug]/[lesson]/+layout.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
```

Comment:
So the layout will be at the lesson level, not course?

Reply:
i think we will have different layouts for both

Reply:
No worries. Can you sketch both layouts and share how you'd like to manage the logic?